api/rest/restapi_test.go

```diff
@@ -686,12 +693,32 @@ func TestAPIUnpinEndpoint(t *testing.T) {
 		errResp := api.Error{}
 		makeDelete(t, rest, url(rest)+"/pins/"+test.ErrorCid.String(), &errResp)
 		if errResp.Message != test.ErrBadCid.Error() {
-			t.Error("expected different error: ", errResp.Message)
+			t.Errorf(
+				"error message : expected: %s, got: %s, CID: %s\n",
+				test.ErrBadCid.Error(),
+				errResp.Message,
+				test.ErrorCid.String(),
+			)
+		}
+
+		makeDelete(t, rest, url(rest)+"/pins/"+test.NotFoundCid.String(), &errResp)
+		if errResp.Code != http.StatusNotFound {
+			t.Errorf(
```

Comment:
I'm not a fan of the additional boilerplate message code. If the test fails, it takes me to a test line where I can read that `errRespCode != http.StatusNotFound`. The 4 lines of pretty messages below give no additional information, but add lines of code.

Reply:
You want all test messages changed in this PR to what it was earlier
something like 
```
t.Errorf("error message : expected: %s, got: %s\n", test.ErrBadCid, errResp.Message)
```